kernel/aster-nix/src/vm/vmar/mod.rs

```diff
@@ -101,23 +101,6 @@ pub(super) struct Vmar_ {
     parent: Weak<Vmar_>,
 }
 
-impl Drop for Vmar_ {
-    fn drop(&mut self) {
-        if self.is_root_vmar() {
-            self.vm_space.clear();
```

Comment:
Currently, this drop function is used to clear the corresponding page table when reclaiming a child process, so that the `Page` structures already used in the child process's page table can be properly handled. Is there another place that can handle this logic after it's removed from here?

Reply:
The `Page`s will be dropped when dropping the page table of the child process. I checked that the pages will always be dropped so I removed the redundant `Drop` for `Vmar` here. I guess for the child `Vmar` it is not necessary too?

Reply:
Fine, I decided not to change the behavior of the Vmar in this PR. I'll keep it intact.